build.sh

```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/env bash
-
-set -o errexit
-set -o pipefail
-set -o nounset
-
-__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-
-docker build --build-arg TAG=${VERSION:-latest} --tag deephaven-image-streams "${__dir}"
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+docker build --build-arg TAG=${VERSION:-latest} --tag deephaven-image-streams "${__dir}"

```

Comment:
@chipkent Just a general comment for your awareness; this script failed for Jeremiah on WSL, we worked around it by manually running `docker build --tag deephaven-image-streams .` and then `docker-compose up`. I had no issue running it on osx